src/app/modules/panel/panel.component.html

```diff
@@ -11,38 +13,62 @@
 
 <ng-template #panelTitle>
   <div
-    class="ds-l-row ds-u-align-items--top ds-u-justify-content--center ds-u-margin--0 pointer panel"
+    class="ds-l-row ds-u-align-items--top ds-u-margin--0 panel"
     [ngClass]="titleClass"
-    (click)="panelStateChange($event)"
-    (keyup.enter)="panelStateChange($event)"
-    tabindex="0"
-    role="button"
-    [attr.aria-expanded]="expand || openAll"
-    attr.data-auto-id="{{ dataAutoId }}"
   >
-    <ng-container *ngIf="iconPlacement === PanelIconPlacementEnum.LEFT">
-      <ng-container *ngTemplateOutlet="plusIcon"></ng-container>
-      <div class="ds-l-col--auto"></div>
-    </ng-container>
-
-    <div class="ds-l-col--auto ds-u-padding-left--0 title">
-      <ng-container *ngIf="title; else titleComplex">{{
-        titleExpanded && (expand || openAll) ? titleExpanded : title
-      }}</ng-container>
-      <ng-template #titleComplex><ng-content select="ng-container"></ng-content></ng-template>
+    <!-- clickable group of possible icons and title fills most of the outmost row -->
+    <div
+      (click)="panelStateChange($event)"
+      (keyup.enter)="panelStateChange($event)"
+      tabindex="0"
+      role="button"
+      [attr.aria-expanded]="expand || openAll"
+      attr.data-auto-id="{{ dataAutoId }}"
+      class="ds-l-col pointer"
+    >
+      <div class="ds-l-row">
+        <!-- Left Icon -->
+        <div
+          class="ds-l-col--auto ds-u-padding-left--0 ds-u-padding-right--2"
+          *ngIf="iconPlacement === PanelIconPlacementEnum.LEFT"
+        >
+          <ng-container>
+            <ng-container *ngTemplateOutlet="plusIcon"></ng-container>
+          </ng-container>
+        </div>
+        <!-- Title -->
+        <div class="title ds-l-col--auto ds-u-padding-left--0">
+          <ng-container *ngIf="title; else titleComplex">{{
+            titleExpanded && (expand || openAll) ? titleExpanded : title
+          }}</ng-container>
+          <ng-template #titleComplex><ng-content select="ng-container"></ng-content></ng-template>
```

Comment:
I got lost here (very fuzzy on content projection)...how does this work/what does it show?

Reply:
this hurt my head too, but it's a weird and strange way we pull in contents from files that consume this component. Basically anything inside the `<app-panel>` tag in a consuming component that is _also_ inside of an `ng-container` will get pulled in between this `ng-template #titleComplex`

here's an [example](https://qnetgit.cms.gov/Bellese/sde-frontend-data-submission-microservice/blob/master/src/app/modules/sde/components/programs/iqr/measure-card/measure-card.component.html#L19-L52) from SDE